src/pages/groupComparison/GroupComparisonUtils.ts

```diff
@@ -122,4 +125,18 @@ export function getPatientIdentifiers(
 export const ENRICHMENTS_TOO_MANY_GROUPS_MSG = "Can't show enrichments for more than two groups - deselect some from the area at the top of the page.";
 export function ENRICHMENTS_TOO_MANY_STUDIES_MSG(enrichmentsType:string) {
     return `Selected comparison groups span more than one study, so we can't show ${enrichmentsType} enrichments. Please deselect groups from the top of the page, or try a different set of groups.`;
+}
+
+export function getDefaultGroupName(filters:StudyViewFilter) {
+    // TODO: generate describe name from filters
+    return "<DESCRIPTIVE GENERATED NAME FROM FILTERS>";
```

Comment:
@jjgao @alisman do we actually want to build this out? I thought it could be cool to autogenerate "Lung cancer" if you have lung cancer selected, but then I realized it would be useless/cluttered if the selection was any more complicated than that. So in the current implementation I leave the name box empty. Thoughts?

Reply:
@adamabeshouse that would be cool, but yeah if it can be a masters thesis project :D

For virtual study, we just put the time as default.

Reply:
this is the name which would be exposed to the user, so I dont think time would look very nice right? Ill change the uid to be based on the time